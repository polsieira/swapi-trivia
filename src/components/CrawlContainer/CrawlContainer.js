import React from 'react';
import Crawl from 'react-star-wars-crawl';
import 'react-star-wars-crawl/lib/index.css';
import './CrawlContainer.scss';

const CrawlContainer = ({ title, subTitle, text }) => (
  <Crawl
    title={title}
    subTitle={subTitle}
    text={text}
  />
)

export default CrawlContainer;