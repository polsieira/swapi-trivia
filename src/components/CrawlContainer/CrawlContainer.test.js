import React from 'react';
import { shallow } from 'enzyme';

import CrawlContainer from './CrawlContainer';

describe('CharacterContainer', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CrawlContainer
      title={'Epsiode 1000'}
      subTitle={'Revenge of the Revenge'}
      text={'Texty text text'}
    />)
  });

  it('should match the snapshot with all data passed correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });


});