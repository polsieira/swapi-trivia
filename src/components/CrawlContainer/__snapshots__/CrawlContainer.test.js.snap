// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterContainer should match the snapshot with all data passed correctly 1`] = `ShallowWrapper {}`;
